itest/lnd_test.go

```diff
@@ -79,6 +79,8 @@ var (
 
 // TestLightningNetworkDaemon performs a series of integration tests amongst a
 // programmatically driven network of lnd nodes.
+//
+//nolint:paralleltest
```

Comment:
IMO we could disable this for the whole `itest` directory (in the `.golanglint-ci` file) as we control the parallel behaviour ourselves here.

Reply:
make sense, moved it to the yml file.